.github/workflows/ci.yaml

```
@@ -24,6 +24,13 @@ jobs:
           distribution: 'temurin'
           java-version: '17'
           cache: 'gradle'
+      - uses: actions/checkout@v3
+      - uses: bufbuild/buf-setup-action@v1.24.0
```

```suggestion
      - uses: bufbuild/buf-setup-action@v1
```

We can just keep this updated with the latest.